plugin/auto/transfer.go

```diff
@@ -0,0 +1,22 @@
+package auto
+
+import (
+	"github.com/coredns/coredns/plugin/file"
+	"github.com/coredns/coredns/plugin/transfer"
+	"github.com/miekg/dns"
+)
+// Transfer implements Transfer.Transferer
+func (a Auto) Transfer(zone string, serial uint32) (<-chan []dns.RR, error) {
+	// look for exact zone match
```

Comment:
this needs to be done as done here: https://github.com/coredns/coredns/pull/3972/files#diff-f1591d056edea707afdeae51b8dc8b8fR12